src/chi/components/sidebar/sidenav.scss

```diff
@@ -197,10 +198,56 @@ $sidenav-xl-width: 12.5rem;
       }
     }
   }
+
+  &.-collapsed {
+    width: $sidenav-collapsed-width;
+
+    .m-sidenav__list > li {
+      > a {
+        font-size: .8125rem;
```

Comment:
Shouldn't these styles be defined on the small viewport/responsive version of base so the two aren't inconsistent?